keyboards/tada68/keymaps/abishalom/keymap.c

```diff
@@ -0,0 +1,97 @@
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BL 0
+#define _FL 1
+#define _NL 2
+
+#define _______ KC_TRNS
+#define SEMI_FN LT(_FL, KC_SCLN)
```

Comment:
Line 12 here, however, *is* needed. **Do not delete or modify this line.**

Reply:
Got it. Accidentally deleted previously.